homeassistant/components/coinbase/config_flow.py

```diff
@@ -51,6 +51,8 @@ async def validate_api(hass: core.HomeAssistant, data):
             get_user_from_client, data[CONF_API_KEY], data[CONF_API_TOKEN]
         )
     except AuthenticationError as error:
+        message = "invalid API key" if "api key" in str(error) else "invalid API secret"
+        _LOGGER.error("Coinbase rejected API credentials due to an %s", message)
```

Comment:
If the idea is debugging, we should log it as debug 😉

Reply:
The idea is to aid in debugging (see linked issue) but my feeling was logging the reason of API key/secret rejection felt like the higher level since it's a rare event (only ever going to happen during set up). To be honest it's probably the title of the PR that is more at fault here. That said if you have a strong feeling this kinda log entry should be `debug`, I have no problem at all with that so say the word and I'll change it :)

Reply:
Well we either provide necessary feedback in the UI or make debug logging. In my opinion error logs are not a fit here.

If needed, we are not solving the issue...

Reply:
Ok, that's a fair point. I'll take it to the debug and provide more detail in the UI

Reply:
Done